pyproject.toml

```diff
@@ -27,6 +27,7 @@ parameterized = "^0.9.0"
 yara-python = "^4.5.0"
 idstools = "^0.6.5"
 aenum = "^3.1.15"
+boto3 = { version = "^1.35.22", optional = true }
```

Comment:
Should we remove this from here if it's included in the extras?

Reply:
With [extras](https://python-poetry.org/docs/pyproject/#extras) you need to declare the dependency as optional. By default it will not install unless explicitly installed like with `poetry install --extras s3`